x-pack/plugins/beats_management/server/lib/adapters/framework/integration_tests/kibana.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+// file.skip
+
+// @ts-ignore
+import { esTestConfig, kbnTestConfig, OPTIMIZE_BUNDLE_DIR } from '@kbn/test';
+// @ts-ignore
+import * as kbnTestServer from '../../../../../../../../src/test_utils/kbn_server';
+// @ts-ignore
+import { xpackKbnServerConfig } from '../../../../../../../test_utils/kbn_server_config';
+import { PLUGIN } from './../../../../../common/constants/plugin';
+
+import { KibanaBackendFrameworkAdapter } from './../kibana_framework_adapter';
+import { contractTests } from './test_contract';
+let servers: any;
+contractTests('Kibana  Framework Adapter', {
+  async before() {
+    servers = await kbnTestServer.startTestServers({
+      adjustTimeout: (t: number) => jest.setTimeout(t),
+      settings: xpackKbnServerConfig,
+    });
+    // const config = legacyServer.server.config();
```

Comment:
My preference is to not check in commented-out code. That's what Git is for. Not sure if we have a policy around this or not.